tests/test_single_topology_v2.py

```diff
@@ -0,0 +1,314 @@
+import functools
+
+import numpy as np
+import pytest
+from rdkit import Chem
+
+from timemachine.fe import single_topology, utils
+from timemachine.fe.single_topology import setup_dummy_interactions
+from timemachine.ff import Forcefield
+
+
+def test_find_chiral_atoms():
```

Comment:
Missing test: `test_find_chiral_bonds`

Would want to add:
1. minimal examples like "F/C=C/F" and "F\C=C/F" -- assert chiral bonds ==  `{(1, 2)}` (currently passing)

2. out-there examples like `"C([H])(C)=C=C([H])(C)"` -- an example of "chirality without stereocenters" from https://www.colby.edu/chemistry/CH241F/Chapter%204.pdf (currently, `single_topology.find_chiral_atoms` says the 2 methyls are chiral but not the other 3 double-bonded carbons, and `single_topology.find_chiral_bonds` returns the empty set, which seems incorrect? seems like an error should be raised on an input like this)

3. more cases of specifications on rings, e.g. cis vs. trans 1,2-dichlorocyclohexane ("C1CC[C@@H]([C@@H](C1)Cl)Cl" and "C1CC[C@H]([C@@H](C1)Cl)Cl"). find_chiral_bonds returns empty set for both, and find_chiral_atoms returns all ring atoms for both... This seemed counterintuitive at first, but on further thought is fine! (Atom-centered approach doesn't exactly specify the concepts "cis" vs. "trans" here, but specifies a specific cis configuration and a specific trans configuration in "absolute" coordinates...)

Reply:
1) will take a look
2) it depends on if this is a forcefield issue or not - our FF currently reqiures torsions be spanned over contiguous atoms with bond/angles adjacent. I need to check and see if there's even a torsion here spanning either of the `C=C` bonds (which would be undefined anyways). One possible way to deal with this is to define an extended torsion skipping the middle "C"
3) I can add more examples for some heterocycles (aromatic and non-aromatic ones)

Reply:
2.

>I need to check and see if there's even a torsion here spanning either of the C=C bonds (which would be undefined anyways).

There are torsion terms spanning these, but they're assigned force constants of 0 as you would expect!

<img width="525" alt="image" src="https://user-images.githubusercontent.com/5759036/170287224-d59a3dab-e087-4979-adf7-16906316f2cc.png">

(via the type with this pattern `'[*:1]~[*:2]=[#6,#7,#16,#15;X2:3]=[*:4]'`)